index.md

```diff
@@ -31,10 +35,11 @@ This is an overview of all Android versions and their corresponding identifiers
     <td><code>T</code></td>
     <td>
       Tiramisu
-      <sup id="fnref:2"><a href="#fn:2" class="footnote">2</a></sup>
+      <sup id="fnref:3"><a href="#fn:3" class="footnote">3</a></sup>
     </td>
     <td rowspan="2"><i>No data</i></td>
     <td rowspan="2"><i>TBA</i></td>
+    <td rowspan="7" class="green">2022-05</td>
```

Comment:
What extra info can I reduce from this?

I think this would be misleading to base any decisions on. I have a Google device, Android 11, and my security patch is 2020-10.